### PR TITLE
USSD Sim

### PR DESCRIPTION
**Pull request by tristanhcole**
@enjeyw's been hiding a USSD Sim 🎁. So much easier than setting up AT/ngrok everytime.

Also; adding `devtools` directory

### CODE REVIEW

**Comment by enjeyw** [comment]
Our directory is getting pretty clogged with random dev tooling files - wondering if we should create a dev_tools or utils directory or similar.

**Comment by tristanhcole** [comment]
> Our directory is getting pretty clogged with random dev tooling files - wondering if we should create a dev_tools or utils directory or similar.

I think that's a good idea. Adding dev tools to the repo is also a good idea imo

**Comment by tristanhcole** [comment]
TODO: update https://github.com/teamsempo/SempoBlockchain/wiki/AWS-Deployment-Guide once merged in

**Comment by tristanhcole** [comment]
I've added changes so all scripts work with following:
- Program Arguments: `path to virtual env`
- Working Directory: `/SempoBlockchain/devtools`

File: devtools/prepare_aws_deploy.sh [hunk 1]
```diff
@@ -1,14 +1,12 @@
 #!/usr/bin/env bash
 
-bash build_docker.sh
-
-# REPOSITORY_URI=290492953667.dkr.ecr.ap-southeast-2.amazonaws.com/blockchaindemo
+#bash build_docker.sh
```
Comment (tristanhcole) [inline devtools/prepare_aws_deploy.sh:3]: commented?

Comment (enjeyw) [reply devtools/prepare_aws_deploy.sh:3]: errggg good catch - was retrying code and didn't wanna wait for the build

Comment (tristanhcole) [reply devtools/prepare_aws_deploy.sh]: 👍

**Pull request merged**